friendster-frontend/vite-project/src/components/Login.jsx

```diff
@@ -23,6 +23,11 @@ const Login = () => {
       .then((idToken) => {
         // Set the ID token to local storage
         localStorage.setItem("userToken", idToken);
+        // Set expiration time (e.g., 1 hour from now)
+        const expirationTime = new Date().getTime() + 30*10 * 1000;//
```

Comment:
It looks like the token expiration time is set to 5 minutes. I believe you said you wanted to set it to 30 minutes?